lib/shared-method.js

```diff
@@ -472,7 +472,8 @@ SharedMethod.convertArg = function(accept, raw) {
     )) {
     return raw;
   }
-  if (raw === null || typeof raw !== 'object') {
+  if (raw === null || typeof raw !== 'object' ||
+     util.inspect(raw).length > 5000) {
     return raw;
   }
   var data = traverse(raw).forEach(function(x) {
```

Comment:
I guess that we probably should bypass certain types to avoid traversing into `req` and `res`. Maybe only recurse into plain objects/arrays?